Make client_secret optional since it's not always required (#32628)
github/docs: content/apps/creating-github-apps/authenticating-with-a-github-app/refreshing-user-access-tokens.md

   Query parameter | Type | Description
   -----|------|------------
   `client_id` | `string` | **Required.** The client ID for your {% data variables.product.prodname_github_app %}. The client ID is different from the app ID. You can find the client ID on the settings page for your app.

>    `client_secret` | `string` | **Optional.** The client secret for your {% data variables.product.prodname_github_app %}. You can generate a client secret on the settings page for your app. Omit this when the user access token was generated using the device flow.

Reviewer: suggestion to move the requirement into to be together. 

```suggestion
   `client_secret` | `string` | **Required** unless the user access token was generated using the device flow. The client secret for your {% data variables.product.prodname_github_app %}.
```
